### PR TITLE
feat:login-page added

### PR DESCRIPTION
**Pull request by niraj-24-aptrix**
check-New codes

### CODE REVIEW

**Review by jeetesh-nariya — APPROVED** [review]
All things is done

File: login.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  <script src="https://cdn.tailwindcss.com"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<section class="bg-gray-50 dark:bg-gray-900">
+  <div class="flex flex-col items-center justify-center px-6 py-8 mx-auto md:h-screen lg:py-0">
+      <a href="#" class="flex items-center mb-6 text-2xl font-semibold text-gray-900 dark:text-white">
+          <img class="w-8 h-8 mr-2" src="https://flowbite.s3.amazonaws.com/blocks/marketing-ui/logo.svg" alt="logo">
+          Flowbite    
+      </a>
```
Comment (jeetesh-nariya) [inline login.php:15]: why us a tag

Comment (jeetesh-nariya) [reply login.php:15]: any idea

**Review by jeetesh-nariya — COMMENTED** [review]
can give me answser

File: login.php (hunk 1 above)
Comment (niraj-24-aptrix) [reply login.php:15]: yes sir i have a idea

**Pull request merged**